beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -47,9 +47,6 @@ const (
 // blindBlobsBundle holds the KZG commitments and other relevant sidecar data for a builder's beacon block.
 var blindBlobsBundle *enginev1.BlindedBlobsBundle
```

Comment:
Are you skipping this because we'll get rid of blinded blobs soon?

Reply:
Correct. Blinded blobs will go away with the new blob sidecar inclusion format. 
Even if it doesn't go away, the blinded blobs bundle don't require its lifetime to be carried between RPC call because the beacon node relies on the builder/relayer returns the full blob on the second call. The issue that this PR address is merely for local building